docs/source/guides/network_node/staking_guide.rst

```diff
@@ -79,6 +79,8 @@ All staking-related operations done by Staker are performed through the ``nucyph
 +----------------------+-------------------------------------------------------------------------------+
 |  ``mint``            | Mint last portion of reward                                                   |
 +----------------------+-------------------------------------------------------------------------------+
+|  ``merge``           | Merge two stakes into one                                                     |
```

Comment:
Hm - I dont expect to see this diff here - perhaps check the base and history again.

Reply:
It's rebased over `main`, while the current target (TI) hasn't been.